content/800-data-platform/100-accelerate/600-faq.mdx

```diff
@@ -33,11 +33,34 @@ This might be an issue with the TypeScript server not yet recognizing the regene
 
 VS Code should now recognize the `$extends` method.
 
-## What regions are Accelerate available in?
-
-Accelerate runs on Cloudflare's network. Cache hits are served from Cloudflare's 300+ locations. Cache misses can be served from 16 different locations at the time of writing. More will be added in the future.
-
-You can find the regions Accelerate is available here: [https://www.cloudflare.com/network/](https://www.cloudflare.com/network/).
+## What regions are Accelerate's cache nodes available in?
+
+Accelerate runs on Cloudflare's network and cache hits are served from Cloudflare's 300+ locations. You can find the regions where Accelerate's cache nodes are available here: [https://www.cloudflare.com/network/](https://www.cloudflare.com/network/).
+
+## What regions is Accelerate's connection pool available in?
+
+When no cache strategy is specified or in the event of a cache miss, the Prisma Client query is routed through Accelerate's connection pool. Currently, queries can be routed through any chosen region among the 16 available locations.
+
+Currently, the list of available regions are:
+
+- Asia Pacific (Mumbai)
+- Asia Pacific (Seoul)
+- Asia Pacific (Singapore)
+- Asia Pacific (Sydney)
+- Asia Pacific (Tokyo)
+- Canada (Central)
+- Europe (Frankfurt)
+- Europe (Ireland)
+- Europe (London)
+- Europe (Paris)
+- Europe (Stockholm)
+- South America (Sao Paolo)
+- US East (N. Virginia)
+- US East (Ohio)
+- US West (N. California)
+- US West (Oregon)
```

Comment:
don't all of these have region code, like `eu-west-1` or something like this? if so, could we add these codes here?

Reply:
I copy pasted the user friendly region names. 
@nikolasburk With the region codes, how should the format be?

Reply:
I would suggest smth like:

- `eu-west-2` (Mumbai | Asia Pacific)

or

- `eu-west-2` (Mumbai, Asia Pacific)

Reply:
or some other combination of code formatted region, parantheses, commas and pipes 😅

Reply:
```suggestion
- Asia Pacific (Mumbai) - `ap-south-1`
- Asia Pacific (Seoul) - `ap-northeast-2`
- Asia Pacific (Singapore) - `ap-southeast-1`
- Asia Pacific (Sydney) - `ap-southeast-2`
- Asia Pacific (Tokyo) - `ap-northeast-1`
- Canada (Central) - `ca-central-1`
- Europe (Frankfurt) - `eu-central-1`
- Europe (Ireland) - `eu-west-1`
- Europe (London) - `eu-west-2`
- Europe (Paris) - `eu-west-3`
- Europe (Stockholm) - `eu-north-1`
- South America (Sao Paulo) - `sa-east-1`
- US East (N. Virginia) - `us-east-1`
- US East (Ohio) - `us-east-2`
- US West (N. California) - `us-west-1`
- US West (Oregon) - `us-west-2`
```